tests/integration/conftest.py

```diff
@@ -50,8 +50,8 @@ async def deploy(ops_test: OpsTest, charm: str):
                 "namespace-rps-limit": "default:50|test:40",
             },
         ),
-        ops_test.model.deploy(APP_NAME_ADMIN, channel="edge"),
-        ops_test.model.deploy(APP_NAME_UI, channel="edge"),
+        ops_test.model.deploy(APP_NAME_ADMIN, channel="1.23/edge"),
```

Comment:
Similar comment to other PRs, let's have a global var for channels, charm names, etc., so it's easier to change the version when we upgrade.